server/controller/FlightController.js

```diff
@@ -353,6 +356,32 @@ router.get(
   }
 );
 
+// @desc Allows an admin to (re)fetch METAR data for a flight
+// @route POST /flights/admin/fetch-metar
+// @access Only moderators and admins
+router.get(
+  "/admin/fetch-metar/:id",
+  checkParamIsUuid("id"),
+  authToken,
+  async (req, res, next) => {
+    try {
+      if (await requesterIsNotModerator(req, res)) return;
+      const flight = await service.getById(req.params.id);
+      if (!flight) return res.sendStatus(NOT_FOUND);
+      // TODO: Why is it so complicated to get the fixes?
```

Comment:
Because the flight fixes are stored in a GeoJSON Format in the db.
This function syncs the access to the addional information like stats...